.ci/build.sh

```diff
@@ -2,30 +2,19 @@
 set -xueo pipefail
 
 # TODO: make sdist work on all, it currently fails for clash-cosim
```

Comment:
Is this still TODO?

Reply:
Doesn't look like it! This is just copied from `master` though, so I'll fix it in an extra commit.